scripts/pin_the_slow_way.py

```diff
@@ -33,15 +33,15 @@
           'bzip2': 'bzip2 1.0.*',
           'fontconfig': 'fontconfig 2.11.*',
           'freetype': 'freetype 2.6.*',
-          'hdf5': 'hdf5 1.8.17',
+          'hdf5': 'hdf5 1.8.17|1.8.17.*',
           'icu': 'icu 56.*',
           'jpeg': 'jpeg 9*',
           'libnetcdf': 'libnetcdf 4.4.*',
           'libpng': 'libpng >=1.6.21,<1.7',
           'libtiff': 'libtiff 4.0.*',
-          'ncurses': 'ncurses 5.9.*',
+          'ncurses': 'ncurses 5.9*',
```

Comment:
The problem with this case is that there is no `5.9.0` it only has two numbers `5.9`. Taking this to its conclusion, it makes sense to simply say `5.9` if that is what we mean.